src/main/java/edu/cornell/kfs/ksr/businessobject/actions/SecurityGroupActionsProvider.java

```diff
@@ -9,25 +9,32 @@
 import org.kuali.kfs.krad.bo.BusinessObjectBase;
 import org.kuali.kfs.krad.util.KRADConstants;
 import org.kuali.kfs.krad.util.UrlFactory;
+import org.kuali.kfs.sys.KFSConstants;
 import org.kuali.kfs.sys.businessobject.actions.BusinessObjectActionsProvider;
 
 import edu.cornell.kfs.ksr.KSRConstants;
+import edu.cornell.kfs.ksr.businessobject.SecurityGroup;
+import edu.cornell.kfs.ksr.businessobject.SecurityProvisioning;
 
 public class SecurityGroupActionsProvider extends BusinessObjectActionsProvider {
 
     @Override
     public List<Action> getActionLinks(BusinessObjectBase businessObject, Person user) {
         final List<Action> actions = super.getActionLinks(businessObject, user);
-
-        String securityProvisioningUrl = generateSecurityProvisioningUrl();
+        
+        String securityProvisioningUrl = generateSecurityProvisioningUrl((SecurityGroup) businessObject);
         final Action action = new Action(KSRConstants.SECURITY_PROVISIONING_URL_NAME, "GET", securityProvisioningUrl);
         actions.add(action);
```

Comment:
To prevent unauthorized users from being able to view the "Edit Provisioning" link by mistake, this section should be updated to check the user's ability to initiate such documents prior to adding the link. See the BusinessObjectActionsProvider superclass for examples.

Reply:
Great suggestion. Who is this restricted to in cynergy currently?

Reply:
In Cynergy, the "Edit Provisioning" link is restricted to those who have permission to see the regular "edit" link on the Security Group lookup.